Plugins/DotNET/DotNETExports.cpp

```diff
@@ -176,27 +137,48 @@ void DotNET::StackPushVector(Vector value)
 
 void DotNET::StackPushEffect(CGameEffect* value)
 {
-    PushedCount += StackPushGameDefinedStructure(0, value);
+    LOG_WARNING("StackPushEffect is deprecated. Use StackPushGameDefinedStructure instead.");
+    StackPushGameDefinedStructure(0, value);
```

Comment:
There's constants for VM Structures here: https://github.com/nwnxee/unified/blob/master/NWNXLib/API/Constants/VirtualMachine.hpp#L32

You'll have to add sqlquery tho.